td.vue/src/i18n/ar.js

```diff
@@ -47,7 +47,12 @@ const ara = {
         }
     },
     demo: {
-        select: 'حدد نموذج تهديد تجريبي من القائمة أدناه'
+        select: 'حدد نموذج تهديد تجريبي من القائمة أدناه',
+        "New Threat Model": "نموذج تهديد جديد",
+        "Demo Threat Model": "نموذج تهديد تجريبي",
+        "Version 2 New Model": "الإصدار ٢ نموذج جديد",
+        "Version 2 Demo Model": "الإصدار ٢ نموذج تجريبي"
```

Comment:
I am unsure that the translations are correct - could you provide translations only for those languages that you speak natively? Translation programs tend to struggle over technical terms like this